src/EventListener/RequestConfigurationSubscriber.php

```diff
@@ -91,4 +91,15 @@ public function onKernelController(ControllerEvent $event): void
         $request->setRequestFormat('json');
         $request->attributes->set(self::ATTRIBUTE_API, true);
     }
+
+	private function getAnnotation(\ReflectionMethod $reflection, string $annotationClass)
```

Comment:
May be it will be better to separate this method into dedicated class? Because it might be useful in `Condenast\BasicApiBundle\ApiDoc\ApiRouteDescriber`, which is not updated yet.

Reply:
ala, reader wrapper? 
Good idea, thks